src/components/Reusable/Icon.js

```diff
@@ -0,0 +1,10 @@
+
+const Icon = (props) => {
```

Comment:
You can use destructuring here. Like `{id, classes, children}` instead of `props`

Reply:
thanks. i have made the changes.